tests/run.sh

```diff
@@ -169,6 +180,12 @@ fi
 
 export TENANTADM_STRIPE_API_KEY=$STRIPE_API_KEY
 
+if [ -n "$K8S" ]; then
+    aws eks --region $AWS_DEFAULT_REGION update-kubeconfig --name $AWS_EKS_CLUSTER_NAME
```

Comment:
To avoid cluttering the default kube config file `~/.kube/config`, can we set the `KUBECONFIG` environment variable to a local path?